src/main/java/Duke.java

```diff
@@ -1,10 +1,146 @@
+                System.out.println("Kamxia. Hope to see you again soon!");
+                System.out.println("____________________________________________________________");
+                break;
+            } else if (userInput.equalsIgnoreCase("list")) {
+                displayTasks();
+            } else if (userInput.startsWith("mark ")) {
+                markTask(userInput);
+            } else if (userInput.startsWith("unmark ")) {
+                unmarkTask(userInput);
+            } else if (userInput.startsWith("todo ")) {
+                addTodoTask(userInput.substring(5));
+            } else if (userInput.startsWith("deadline ")) {
+                String[] split = userInput.substring(9).split(" /by ");
+                AddDeadlineTask(split[0], split[1]);
+            } else if (userInput.startsWith("event ")) {
+                String[] split = userInput.substring(6).split(" /from | /to ");
+
+                if (split.length >= 3) {
+                    AddEventTask(split[0], split[1], split[2]);
+                } else {
+                    System.out.println("Invalid input format for event task.");
+                }
+            } else {
+                addTask(userInput);
+            }
+        }
+
+        scanner.close();
+    }
+
+    private static void displayTasks() {
+        System.out.println("____________________________________________________________");
+        System.out.println("LaiLaiLai, Here are the tasks in your list:");
+        for (int i = 0; i < taskCount; i++) {
+            System.out.println((i + 1) + "." + tasks[i]);
+        }
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void addTask(String description) {
+        if (description.isEmpty()) {
+            System.out.println("Anyhow one...write your task properly leh :(");
+            return;
+        }
+        tasks[taskCount++] = new Task(description);
+        System.out.println("____________________________________________________________");
+        System.out.println("Added: " + description);
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTodoTask(String description) {
+        if (description.isEmpty()) {
+            System.out.println("Anyhow one...write your task properly leh :(");
+            return;
+        }
+        tasks[taskCount++] = new Todo(description);
+        System.out.println("____________________________________________________________");
+        System.out.println("Got it. I've added this task:");
+        System.out.println("  " + tasks[taskCount - 1]);
+        System.out.println("Now you have " + taskCount + " tasks in the list.");
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void AddDeadlineTask(String description, String by) {
```

Comment:
methods "AddDeadlineTask" and "AddEventTask" are correctly named using verbs but should start with a lowercase character